src/Lesson5/HomeTask6.java

```diff
@@ -0,0 +1,12 @@
+package Lesson5;
+
+public class HomeTask6 {
+
+
+    public class NewClas extends Car {
+
+public void main(String[] args) {
+
+}
```

Comment:
6. Take the classes from the task 5. Create a new test class.
Import everything (*) from your created packages.
Now try to create objects of your classes without specifying the packages in front of the class names.
 What happened? How can we avoid this?